openshift/vault-manager.template.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: vault-manager
+objects:
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    labels:
+      app: vault-manager
+    name: vault-manager
+  spec:
+    replicas: ${{REPLICAS}}
+    strategy:
+      type: RollingUpdate
+      rollingUpdate:
+        maxUnavailable: 0
+        maxSurge: 1
+    selector:
+      matchLabels:
+        app: vault-manager
+    template:
+      metadata:
+        labels:
+          app: vault-manager
+      spec:
+        containers:
+        - image: ${IMAGE}:${IMAGE_TAG}
+          imagePullPolicy: Always
+          name: vault-manager
+          command: ["/bin/sh"]
+          args: 
+          - "-c"
+          - "while true; do /vault-manager; sleep 15m; done;"
+          env:
+          - name: GRAPHQL_QUERY_FILE
+            value: ${GRAPHQL_QUERY_FILE}
+          - name: GRAPHQL_SERVER_URL
+            valueFrom:
+              secretKeyRef:
+                key: gql.base_url
+                name: ${GRAPHQL_SERVER_URL}
```

Comment:
`name` is the Secret's name, we can probably just hard code it to be `vault-manager`? or something like VAULT_MANAGER_SECRET_NAME if you prefer that the secret name is parameterized?